Makefile

```diff
@@ -28,9 +28,9 @@ endif
 server: server/.depensure
 ifneq ($(HAS_SERVER),)
 	mkdir -p server/dist;
-	cd server && env GOOS=linux GOARCH=amd64 $(GO) build -o dist/plugin-linux-amd64;
-	cd server && env GOOS=darwin GOARCH=amd64 $(GO) build -o dist/plugin-darwin-amd64;
-	cd server && env GOOS=windows GOARCH=amd64 $(GO) build -o dist/plugin-windows-amd64.exe;
+	cd server && CGO_ENABLED=0 env GOOS=linux GOARCH=amd64 $(GO) build -o dist/plugin-linux-amd64;
```

Comment:
Looks good!
But I don't know why `CGO_ENABLED=0` is out of `env`. Is there a reason?

Reply:
Not really. I moved it into `env`

Reply:
👍